packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.constants.js

```diff
@@ -1,3 +1,26 @@
+export const MONITORING_STATUSES = {
+  DISABLED: 'disabled',
+  NOPROACTIVE: 'no-proactive',
+  PROACTIVE: 'proactive',
+};
+
+export const DC_2_ISO = {
+  BHS: 'CA',
+  DC: 'CA',
+  ERI: 'GB',
+  GRA: 'FR',
+  GSW: 'FR',
+  HIL: 'US',
+  LIM: 'DE',
+  P: 'FR',
```

Comment:
**question:** Are you sure this is the right DC?

Reply:
https://github.com/ovh/manager/pull/9520/files#r1266676420